sample/QueueTrigger-Powershell/queueTrigger.ps1

```diff
@@ -1,4 +1,7 @@
 $in = Get-Content $triggerInput
 $json = $in | ConvertFrom-Json
+
+Write-Output "PowerShell script processed queue message '$json'"

```

Comment:
Not related to these changes, but I wanted to add an example of a PS console write (we didn't have any in samples)